IBAnimatable/Animatable.swift

```diff
@@ -198,7 +198,7 @@ public extension Animatable where Self: UIView {
 
   }
   
-  public func moveTo(x: Double, y: Double, completion: AnimatableCompletion? = nil) {
+  public func moveTo(_ x: Double, y: Double, completion: AnimatableCompletion? = nil) {
```

Comment:
Shall we keep the "x:" part like `public func moveTo(x: Double, y: Double, completion: AnimatableCompletion? = nil)` or we can use `public func moveToX(_ x: Double, y: Double, completion: AnimatableCompletion? = nil)` ?

I think the first one is more Swift3ify.